src/exports.rs

```diff
@@ -1,6 +1,9 @@
-// api.rs includes the public wasm API
-// when included, this whole file should be wrapped by
-// #[cfg(target_arch = "wasm32")]
+#![cfg(target_arch = "wasm32")]
+
+///! exports exposes the public wasm API
+///! allocate and deallocate should be re-exported as is
+///! do_init and do_wrapper should be wrapped with a extern "C" entry point
+///! including the contract-specific init/handle function pointer.
```

Comment:
These should be `//!` to be doc comments for this file, `///` is for the thing below it

Reply:
Ah, thanks